invisible_cities/reco/icaro_components_test.py

```diff
@@ -0,0 +1,136 @@
+import pytest
+
+import numpy          as np
+import numpy.testing  as npt
+import pandas         as pd
+import pandas.testing as pdt
+
+from hypothesis             import given
+from hypothesis.strategies  import floats
+
+from .. reco                import icaro_components as icarcomp
+from ..types.symbols        import KrFitFunction
+from .. evm.ic_containers   import FitFunction
+
+
+@given(floats(min_value = 0,
+              max_value = 10),
+       floats(min_value = 1000,
+              max_value = 1600),
+       floats(min_value = 1e3,
```

Comment:
```suggestion
       floats(min_value = 1e2,
```
let's test also this other case